FluidNC/src/xmodem.cpp

```diff
@@ -50,35 +51,6 @@ static void _outbytes(uint8_t* buf, size_t len) {
     serialPort->write(buf, len);
 }
 
-/* CRC16 implementation acording to CCITT standards */
```

Comment:
In case anyone wonders why i decided to extract this to its own file:
I didn't want to add a redundant implementation for CRC16 if there is already a perfectly working copy available.

Reply:
Instead of just calling it "crc.cpp", how about identifying the variant, i.e. "crc16_ccitt.cpp"

Reply:
Done